backend/src/main/java/com/eshka/controller/TeamController.java

```diff
@@ -46,6 +46,13 @@
                 HttpStatus.OK);
     }
 
+    @ApiOperation("join team")
+    @GetMapping("/{id}")
```

Comment:
Here should be another path. Something like that:  `@GetMapping("/join-link/{id}")`